### PR TITLE
Added an action button to change your entire chameleon outfit at once

### PR DESCRIPTION
**Pull request by Cruix**
:cl:
add: Added a button to chameleon items to change your entire chameleon outfit to the standard outfit of a selected job.
/:cl:

The extra button appears when you first equip a chameleon item, and disappears when your remove the last chameleon item you are holding. If you pick up additional chameleon items, the button will not be duplicated.

[why]: # 90% of the time, chameleon items are used to emulate the outfit of a particular job, and this is far more convenient than having to scroll through the gigantic, unsorted menu for each individual item to find the names of the correct items.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Nice

File: code/modules/clothing/chameleon.dm [hunk 1]
```diff
@@ -68,6 +68,69 @@
 	return 1
 
 
+/datum/action/chameleon_outfit
+	name = "Select Chameleon Outfit"
+	button_icon_state = "chameleon_outfit"
+	var/list/outfit_options //By default, this list is shared between all instances.\
+							It is not static because if it were, subtypes would not be able to have their own.\
+							If you ever want to edit it, copy it first.
```
Comment (SpaceManiac) [inline code/modules/clothing/chameleon.dm]: >`\`

Please don't do this to me.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
Great feature, should make for a lot of fun. Not OP because the outfit is not necessarily perfect, but will help traitors not forget really easy stuff. Just some nitpicks:

* Please do not backslash in comments
* "Shaft Miner (Lavaland + Equipment)" should include the explorer's hood
* Jobs with stamps should include their stamps
* I think the action button animates a little too rapidly

**Comment by Cruix** [comment]
Had to rewrite stamps so that they are actual chameleon items.
Apparently hooded suits and hardsuits are not related, they just use nearly identical code with slightly different proc and var names. I might fix that later.
The action button icon changing quickly is the whole point.

**Review by SpaceManiac — APPROVED** [review]
Personally I find it a little bit visually distracting but that's just a matter of opinion. Looks good to go.

**Pull request merged**